src/Core/src/ImageSources/FileImageSourceService/FileImageSourceService.iOS.cs

```diff
@@ -41,7 +41,7 @@ public partial class FileImageSourceService
 			{
 				Logger?.LogWarning(ex, "Unable to load image file '{File}'.", imageSource.File);
 
-				throw;
```

Comment:
@mattleibow I understand that throwing the exception at this point was a expected behavior, right?